### PR TITLE
IP-182 | Initial commit

### PR DESCRIPTION
**Pull request by Pradnya-glitch**

### CODE REVIEW

File: infra/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+service: testing-app
```
Comment (vitulrana) [inline infra/serverless.yml:1]: It should be `service: testingService`

File: infra/serverless.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+custom:
+  host:
+    ${opt:host, 'local'}
+  profile:
+    ci:
+    local: medly-${opt:stage}
+  testingApiEnv:  ${ssm:/aws/reference/secretsmanager/testingApiEnv~true}
```
Comment (vitulrana) [inline infra/serverless.yml:9]: I don't think we need to create secret as of now.

File: infra/serverless.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        timeout: 20
+        events:
+            - http:
+                  path: /
+                  method: ANY
+                  cors: true
+            - http:
+                  path: /{proxy+}
+                  method: ANY
+                  cors: true
```
Comment (vitulrana) [inline infra/serverless.yml:59]:
```suggestion
        events:
            - http:
                  path: /{proxy+}
                  method: ANY
                  cors: true
```

**Review by vitulrana — CHANGES_REQUESTED** [review]
@Pradnya-glitch App does not have Okta Setup as of now but Backend Service is using Okta Setup. Should we remove Authorization stuff from here ?

**Comment by Pradnya-glitch** [comment]
> @Pradnya-glitch App does not have Okta Setup as of now but Backend Service is using Okta Setup. Should we remove Authorization stuff from here ?

Going ahead we might add it in app..as we want to use it for complete testing

**Review by vitulrana — APPROVED** [review]

File: .github/workflows/build.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+              run: yarn spec:lint
+
+            - name: Create bundle
+              run: yarn dist
+
+            - name: Upload dist
+              uses: actions/upload-artifact@v2
+              with:
+                  name: dist
+                  path: dist
```
Comment (niraj8) [inline .github/workflows/build.yaml:38]: This step is not needed, please remove this

**Pull request merged**